server/anoixo-server/text_providers/test_Nestle1904LowfatProvider.py

```diff
@@ -90,6 +90,57 @@ def raise_exception():
     assert basex_session_mock.return_value.close.call_count == 3
 
 
+def test_build_query_string_adds_extra_attributes(mocker, basex_session_mock, provider):
+    basex_query_spy = mock_basex_on_query_execute(mocker, basex_session_mock, lambda: '[]')
+    query = TextQuery({
```

Comment:
this TextQuery can be simpler since it's not important to the test. I suggest removing the empty wordquery (marked with "# no attributes") and collapsing the whole thing